subtests/docker_cli/systemd/p4321.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=docker-autotest systemd subtest. If you see this file on a live system it means something has failed very badly
+After=docker-latest.service
+
+[Service]
+ExecStart=/usr/bin/docker-latest run -p 4321:4321 --name p4321 p4321
+ExecStop=-/usr/bin/docker-latest stop p4321
+ExecStop=/usr/bin/docker-latest rm -f p4321
```

Comment:
Ooops, yes, definitely what @cevich said. Since the `docker build` is done in the test, the `rmi` should be as well.